File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        booleanArray = [] 
        max_candies= max(candies)
        for i in range(len(candies)):
            if candies[i]+ extraCandies>=max_candies:
                booleanArray.append(True)
            else:
                 booleanArray.append(False) 
        return   booleanArray        
        

File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
    List<Boolean> booleanArray  = new ArrayList<>();
      int max=Arrays.stream(candies).max().getAsInt();
        for(int i=0;i<candies.length;i++)
        {
            if(candies[i]+extraCandies>=max)
            {
                booleanArray.add(true);
            }
            else{
                booleanArray.add(false);
            }
        }
        return booleanArray;
    }
   
}